casadm/extended_err_msg.c

```diff
@@ -182,6 +182,11 @@ struct {
 		OCF_ERR_STANDBY_ATTACHED,
 		"Cache device is attached, please detach it first with --standby --detach"
 	},
+	{
+		OCF_ERR_CORE_SIZE_MISMATCH,
+		"Core volume size does not match with"
```

Comment:
"to match something" (without with) is grammatically correct

Reply:
corrected